web/tests/Web.E2ETests/Features/School/HistoricData.feature

```diff
@@ -137,17 +137,17 @@
           | 2018 to 2019 |       |                               |                                    |
           | 2019 to 2020 |       |                               |                                    |
           | 2020 to 2021 |       |                               |                                    |
-          | 2021 to 2022 | 350   |                               |                                    |
+          | 2021 to 2022 | 350   | 310.73                        | 308.02                             |
 
     @HistoricalTrendsFlagEnabled
```

Comment:
do we need this tag now?

Reply:
I was going to suggest we keep it in just in case the feature needs to be disabled again for some reason